src/analysis/retail/evoker/augmentation/modules/normalizers/EbonMightNormalizer.ts

```diff
@@ -20,6 +30,9 @@ class EbonMightNormalizer extends EventsNormalizer {
     const fixedEvents: any[] = [];
     events.forEach((event: AnyEvent, idx: number) => {
       const linkedEvents = event._linkedEvents?.find((x) => x.relation === EBON_MIGHT_BUFF_LINKS);
+      const removeApplyLinks = event._linkedEvents?.find(
+        (x) => x.relation === EBON_MIGHT_APPLY_REMOVE_LINK,
+      );
```

Comment:
Maybe use `GetRelatedEvents` instead

Reply:
Changed to HasRelatedEvent, defo cleaner solution.